pkg/agent/manager/sync.go

```diff
@@ -62,26 +59,37 @@ func (m *manager) synchronize(ctx context.Context) (err error) {
 			outdated++
 		default:
 			// SVID is good
-			return
+			return false
 		}
-		// we've exceeded the CSR limit, don't make any more CSRs
-		if len(csrs) < node.CSRLimit {
+
+		return true
+	})
+
+	// TODO: this values are not real, we may remove
```

Comment:
What do you mean by this? Why not remove it now? A clearer comment would be good.

Reply:
that is to start a discussion :D, basically we are adding a telemetry, and log than say we are rotating X entries becuase expiration and Y because we mark them as outdated (because it changes its ttl or dns) but really we are only updating 500, in current call the next ones keep as stale, so if we have 500 entries to update because `rotation` and 200 more because we `outdated`, we really are updating 500 (node.CSRLimit).
So we can say we add X and Y but not we are really updating them `now`. Make it sense?